### PR TITLE
Closes #1327: Added a new block display to Events view for past occurrences.

### LINKED ISSUE #1327: Frequently Requested Customization | Past Events View
## Problem/Motivation
In my short time here, the request I have heard from different clients is to be able to quickly showcase past events.  Our current solution (here, use the calendar widget) is never what people are looking for w/r/t past events.

### Describe the bug
Customer story: People are proud of their events, and they usually have more in the past than in the future.  They want to be able to show their audience “look at what we’ve done!” - the calendar widget is more of a way go hunting for past events, not a way to display them.

### To Reproduce
A good example of a hard coded "past events page" created by a client: https://uxua.arizona.edu/past-events

## Proposed resolution
Create a view for News that shows past events from most recent to most distant.

### PR DESCRIPTION
**Pull request by maine-inventor**
## Description

Added a 'Past Events' views display withing the existing Calendar view
## Related issues

Closes #1327 

## How to test

[ ] After enabling the AZ Demo content module, update a few of the AZ Events nodes' dates to have occurred prior to todays date: /admin/content?title=&type=az_event&status=All&langcode=All

[ ] Create a flexible page: node/add/az_flexible_page
[ ] Add a new Page Element of type View 
[ ] Add a Title
[ ] Dropdown set "View*" = "AZ Events"
[ ] Dropdown set "Display*" = "Grid Past with Images"
[ ] Save

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [x] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_event/config/install/views.view.az_events.yml [hunk 1]
```diff
@@ -14,6 +14,8 @@ dependencies:
     - taxonomy
     - user
     - views_bootstrap
+_core:
+  default_config_hash: '-z4hMwNWaeV3NT77gAud352wb1_wIBGPhovFwfARevo'
```
Comment (tadean) [inline modules/custom/az_event/config/install/views.view.az_events.yml:18]: This config hash should be removed.
```suggestion
```

Comment (joeparsons) [inline modules/custom/az_event/config/install/views.view.az_events.yml:18]:
```suggestion
```

File: modules/custom/az_event/config/install/views.view.az_events.yml [hunk 2]
```diff
@@ -736,7 +884,6 @@ display:
         - 'languages:language_interface'
         - url
         - url.query_args
-        - user
         - 'user.node_grants:view'
         - user.permissions
-      tags: {  }
+      tags: {  }
```
Comment (joeparsons) [inline modules/custom/az_event/config/install/views.view.az_events.yml:889]:
```suggestion
      tags: {  }

```

**Review by joshuasosa — APPROVED** [review]
Useful addition. My group has usually used the Calendar layout for displaying past events, but I think the grid with images look is fine.

**Review by tadean — APPROVED** [review]

**Pull request merged**